components/match2/wikis/overwatch/match_summary.lua

```diff
@@ -12,19 +12,20 @@ local MapModes = require('Module:MapModes')
 local OpponentDisplay = require('Module:OpponentDisplay')
 local VodLink = require('Module:VodLink')
 
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
```

Comment:
Base is being imported, but not used.

Look at eg. the implementation of R6, Valorant, Mobile Legends, Splitgate or Wild Rift